types/iframe-resizer/iframe-resizer-tests.ts

```diff
@@ -1,9 +1,10 @@
-import iframeResizer = require("iframe-resizer");
+import { iframeResizer } from 'iframe-resizer';
+import ifr = require("iframe-resizer");
```

Comment:
`ifr` should still be callable, so (1) there should be a test of that (2) you'll still need all the previous code in index.d.ts.

Reply:
@sandersn - I think this is what you were referring to. This is the combination I could find that would allow `ifr` to be callable and still allow the `import { iframeResizer } from 'iframe-resizer';` version to work as expected.